src/conf.py

```diff
@@ -82,6 +82,7 @@
     'augur': ('https://docs.nextstrain.org/projects/augur/page/', None),
     'auspice': ('https://docs.nextstrain.org/projects/auspice/page/', None),
     'cli': ('https://docs.nextstrain.org/projects/cli/page/', None),
+    'docs.nextstrain.org': ('https://docs.nextstrain.org/page/', None),
```

Comment:
Took me a second to realize this needed to be added because the [augur/filter-and-subsampling.rst](https://github.com/nextstrain/augur/blob/b0c5010ec16744dbf8e51225b023d5ed1bc18ff6/docs/usage/cli/snippets/filtering-and-subsampling.rst?plain=1#L134) refers to this with `:doc:`

Reply:
Added clarity in 96b3b9598105dc39a6f118cb752df606747d7191

Reply:
This will likely get weird at some point, as self-references like this means the references always use the published version, not the to-be-published version being built.

Reply:
Alternative proposed in https://github.com/nextstrain/augur/pull/1431#issuecomment-2067315935